src/unleashclient.cpp

```diff
@@ -61,12 +68,19 @@ void UnleashClient::initializeClient() {
         // Initial fetch of feature flags
         auto apiFeatures = m_apiClient->features();
         if (apiFeatures.empty()) {
-            std::cerr << "Attempted to initialize an Unleash Client instance "
-                         "without server response."
-                      << std::endl;
-            return;
+            std::cerr << "Attempted to initialize an Unleash Client instance without server response." << std::endl;
+            std::ifstream cacheFile(m_cacheFilePath, std::fstream::in);
```

Comment:
I think we could avoid to load the file if the `cacheFile` is not set or disable the "Could not open cache file" comment at least.